integration/daemon/daemon_test.go

```diff
@@ -378,3 +381,57 @@ func TestDaemonProxy(t *testing.T) {
 		assert.Assert(t, !strings.Contains(string(logs), userPass), "logs should not contain the non-sanitized proxy URL: %s", string(logs))
 	})
 }
+
+func TestLiveRestore(t *testing.T) {
+	skip.If(t, runtime.GOOS == "windows", "cannot start multiple daemons on windows")
+
+	t.Run("volume references", testLiveRestoreVolumeReferences)
+}
+
+func testLiveRestoreVolumeReferences(t *testing.T) {
```

Comment:
Was there a specific reason for splitting this into two functions? (I can see the reason for the subtests _inside_ `testLiveRestoreVolumeReferences()`, but was a bit confused about the split here 😅 🤔

Reply:
Potential other live restore tests.
Just namespacing.